### PR TITLE
Fix Android Studio ARTK SDK Example app projects

### PR DESCRIPTION
**Pull request by BoiseBaked**
Lots of repetitive changes blah. I had to fix build one syntax error causing ndk-build to not succeed - see change in ".../lib/SRC/ARWrapper/ARController.cpp" at the bottom of this change set.

### CODE REVIEW

File: lib/SRC/ARWrapper/ARController.cpp [hunk 1]
```diff
@@ -873,7 +873,7 @@ bool ARController::stopRunning()
 	}
 
 #if HAVE_NFT
-    mTrackerWasUpdated = false;
+    //mTrackerWasUpdated = false;
```
Comment (philip-lamb) [inline lib/SRC/ARWrapper/ARController.cpp]: Can you expand on why this change was required?

Comment (BoiseBaked) [inline lib/SRC/ARWrapper/ARController.cpp]: Hey Philip

Because the containing source wasn't building. The compiler was complaining
that the field "mTrackerWasUpdated" was undefined/undeclared. I scanned all
of the headers and source files in the code base (I think) for
mTrackerWasUpdated and I came up with nothing.

So I have no idea why the source file was checked-in into master that way.

Do you?
JWolf

Comment (BoiseBaked) [inline lib/SRC/ARWrapper/ARController.cpp]: Just sync-up to the latest and greatest "master" branch from the public
ARTK5 github repo, in a clean local repo, and do a "./build.sh" in
"android".

JWolf

Comment (philip-lamb) [inline lib/SRC/ARWrapper/ARController.cpp]: OK, this line somehow appeared in a merge from a different branch where the variable mTrackerWasUpdated actually exists. Let's remove the line altogether, and then I think we're good to merge.
